sky/clouds/service_catalog/images/provisioners/cuda-azure-grid.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+sudo apt update
+sudo apt install -y build-essential
+
+echo "Installing GRID driver..."
+GRID_DRIVER_URL="https://download.microsoft.com/download/8/d/a/8da4fb8e-3a9b-4e6a-bc9a-72ff64d7a13c/NVIDIA-Linux-x86_64-535.161.08-grid-azure.run"
+GRID_DRIVER_FILE="NVIDIA-Linux-x86_64-535.161.08-grid-azure.run"
+
+wget -nv $GRID_DRIVER_URL -O $GRID_DRIVER_FILE
+sudo chmod +x $GRID_DRIVER_FILE
+sudo sh $GRID_DRIVER_FILE --silent
```

Comment:
```suggestion
sudo sh $GRID_DRIVER_FILE --silent --disable-nouveau
```
Should we disable nouveau driver? It seems there will be a significant performance difference for oss vs enterprise driver.